tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>Overall comments:</p><ol><li>This is expected to be a tutorial. Put it under <code>docs/tuotrial/Basic</code> , and update the corresponding documentation index page such that it can be correctly displayed in docs. </li><li class="ql-indent-1">Typically, the Jupyter notebook will be executed automatically by the CI system and displayed in the website. See http://www.open3d.org/docs/latest/tutorial/Basic/pointcloud.html as an example.</li><li class="ql-indent-1">Howerver, in this case, our CI currently doesn't support CUDA. Therefore:</li><li class="ql-indent-2">Clean all outputs -&gt; run all cells again -&gt; save the executed notebook -&gt; commit the changes</li><li class="ql-indent-2">Exclude the <code>make_docs.py</code>  from running this notebok. See <a href="https://github.com/intel-isl/Open3D/blob/master/docs/make_docs.py#L373-L397" target="_blank">https://github.com/intel-isl/Open3D/blob/master/docs/make_docs.py#L373-L397</a>. Consider adding a list of "ignored" jupyter notebooks and remove all such notebooks from <code>nb_paths</code>.</li><li>Rename this file to just <code>tensor.ipynb</code>.</li><li>Like other tutorials, write more discriptions as text sections in Jupyter. E.g. what is a tensor, what can a tensor do, why it is used in Open3D, etc. You may look at <a href="https://pytorch.org/docs/stable/tensors.html" target="_blank">https://pytorch.org/docs/stable/tensors.html</a> and <a href="https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html" target="_blank">https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html</a> as examples.</li><li>Make things consistent. For example, add <code>.</code>  to the end of all comments.</li></ol><p> </p><p>Specific comment to this cell:</p><ol><li>Let's not import torch here, import it at the cell where DLPack is used. </li><li><code>import open3d.core as o3c</code>  similar to <a href="https://github.com/intel-isl/Open3D/blob/master/python/test/core/test_tensorlist.py#L28" target="_blank">https://github.com/intel-isl/Open3D/blob/master/python/test/core/test_tensorlist.py#L28</a> can shorten the code.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>